src/test/java/hw2/ex2/SecondScenarioTest.java

```diff
@@ -0,0 +1,278 @@
+        // display (we'll not use it in the further steps)
+        WebElement actualLogRowsWebElement = driver.findElement(By.
+                className("info-panel-section"));
+        List<String> actualLogRowsValues = Arrays.
+                asList(actualLogRowsWebElement.getText().split("\n"));
+        sa.assertEquals(actualLogRowsValues.size(), 2);
+        sa.assertTrue(actualLogRowsWebElement.getText().
+                contains("Wind") && actualLogRowsWebElement.getText().
+                contains("true"));
+        sa.assertTrue(actualLogRowsWebElement.getText().
+                contains("Water") && actualLogRowsWebElement.getText().
+                contains("true"));
+
+        // 13. Select radio
+        // There we should click on radio with label
+        // "Selen" and check if it is checked
+        driver.findElement(By.xpath("//div[3]//label[4]")).click();
+        WebElement selenRadioButton = driver.findElement(By.
+                xpath("//div[3]//label[4]//input[1]"));
+        assertTrue(selenRadioButton.isSelected());
+
+        // 14. Assert that for radiobutton there is
+        // a log row and value is corresponded to
+        // the status of radiobutton.
+        // There we we check if the panel with information
+        // really contains such info as "Selen"
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if info about theis radio doesn't
+        // display (we'll not use it in the further steps)
+        sa.assertTrue(actualLogRowsWebElement.getText().contains("Selen"));
+
+        // 15. Select in dropdown
+        // There we should get the 'select' element with help of
+        // Select-class. Then we select "Yellow" here and check
+        // if the checked option is really "Yellow"
+        Select selectColorDropDown = new Select(driver.findElement(By.
+                xpath("//select[@class='uui-form-element']")));
+        selectColorDropDown.selectByVisibleText("Yellow");
+        WebElement actuallySelected = selectColorDropDown.
+                getFirstSelectedOption();
+        assertEquals(actuallySelected.getText(), "Yellow");
+
+        // 16. Assert that for dropdown there is
+        // a log row and value is corresponded
+        // to the selected value.
+        // There we we check if the panel with information
+        // really contains such info as "Yellow"
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if info about this selection doesn't
+        // display (we'll not use it in the further steps)
+        sa.assertTrue(actualLogRowsWebElement.getText().contains("Yellow"));
+
+        // 17. Unselect and assert checkboxes
+        // There we should click on checkboxes with labels
+        // "Water" and "Wind" again and check if they are unchecked
+        driver.findElement(By.
+                xpath("//div[@class='main-content']//div[2]//label[1]")).
```

Comment:
Xpath does not readble

Reply:
Solved: I changed this xpath on my previous list of checkbox labels, now it's more readable